scilpy/tractograms/dps_and_dpp_management.py

```diff
@@ -63,9 +89,62 @@ def project_map_to_streamlines(sft, map_volume, endpoints_only=False):
     return streamline_data
 
 
-def perform_streamline_operation_per_point(op_name, sft, dpp_name='metric',
-                                           endpoints_only=False):
-    """Peforms an operation per point for all streamlines.
+def project_dpp_to_map(sft, dpp_key, sum_lines=False, to_endpoints=False):
+    """
+    Saves the values of data_per_point keys to the underlying voxels. Averages
+    the values of various streamlines in each voxel. Returns one map per key.
+    The streamlines are not preprocessed here. You should probably first
+    uncompress your streamlines to have smoother maps.
+
+    Parameters
+    ----------
+    sft: StatefulTractogram
+    dpp_key: str
+        The data_per_point key to project to a map.
+    sum_lines: bool
```

Comment:
It would be clearer if it was an option to normalise map instead, I think :)

Reply:
Hmm, ok. But the two options here are to sum or to average. It's not really the same as normalizing, which I'm not even sure what it would mean here. What do you want?

Reply:
Good keep it that way